openapi.yml

```diff
@@ -334,6 +334,40 @@ components:
           type: "array"
           items:
             type: "string"
+
+        owner:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        manufacturer:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        model:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        instrument_type:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        measured_variable:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        available_date:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        decommission_date:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        related_identifier:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        alternative_identifier:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        instrument_resource_type:
+          type: "string"
+          example: "Classifying Independent Hybridity"
+        funding_reference:
+          type: "string"
+          example: "Classifying Independent Hybridity"
```

Comment:
Do we care about these examples? If so, should they all be the same?

Reply:
I believe we do, but I still need to touch base with Briana about what those values would actually be

Reply:
@ajkiessl do you know if we want to have examples here? Looking further, it looks like some have them & some don't

Reply:
I think these will be fine without examples. If anyone is using our API for uploading instruments, they'll probably know what these fields should be, or they'll at least be in close contact with us if they need to know.